design/FY2019/Interzone Air Wall NFP and Design.md

```diff
@@ -0,0 +1,192 @@
+
+[Justification for New Feature](#justification-for-new-feature)
+
+[E-mail and  Conference Call Conclusions](#e-mail-and-conference-call-conclusions)
+
+[Approach](#approach)
+
+[Testing/Validation/Data Sources](#testingvalidationdata-sources)
+
+[Input Output Reference Documentation](#input-output-reference-documentation)
+
+[Engineering Reference](#engineering-reference)
+
+[Example File and Transition Changes](#example-file-and-transition-changes)
+
+[References](#references)
+
+[Design](#design)
+
+## Justification for New Feature ##
+
+Open boundaries between thermal zones occur frequently in buildings. Examples include:
+ - Open plan offices with core and perimeter zones.
+ - Open doorway from one space to another, e.g. hallway opening into a larger space.
+ - Service opening, e.g. serving window between kitchen and dining area.
+ - High spaces where the occupied zone is conditioned differently from the rest, e.g. atrium or auditorium
+
+Open thermal boundaries allow the following heat transfer to occur:
+ - Solar gains and daylighting to pass through as if there is no boundary
+ - Radiant exhange between building surfaces as if there is no boundary.
+ - Distribute radiant internal  gains as if there is no boundary.
+ - Allow airflow and/or mixing to occur between the adjacent zones
+
+This type of boundary occurs frequently in energy models, and is often called an "air wall." 
+EnergyPlus has no direct way to model such a boundary. Common workarounds include some 
+combination of the following:
+ - Near-perfect interzone window
+ - Surface using `Material:InfraredTransparent`
+ - Surface using a very thin highly conductive material
+ - `ZoneMixing` and `ZoneCrossMixing`
+ - AirflowNetwork openings
+
+All of these options have serious limitations, especially the lack of solar/daylight penetration and 
+no direct radiant exchange across the boundary. 
+
+## E-mail and  Conference Call Conclusions ##
+
+n/a
+
+## Approach ##
+
+The proposed approach is to remove the air wall surface entirely from most of the heat transfer calculations
+and create combined superzones for solar, daylighting, and radiant exchange calculations. Air exchange is
+more complicated.
+
+### Radiant Exchange ###
+For surface-to-surface radiant exchange, all surfaces in the zones connected by airwalls will be combined into
+a superzone for the radiant exchange calculations. Radiant gains from lights, people, HVAC equipment, etc. will 
+be distributed to all surfaces in the superzone.
+
+#### Pitfalls ####
+ - Radiant exchange calculations become very slow for zones with a large number of surfaces. This approach
+ will exaggerate that problem.
+ - Within the superzone it is likely that many surfaces will not have a direct view to all of the surfaces
+ in the combined zone. Presently there is no algorithm to exclude certain views. View factors could be determined
+ external to EnergyPlus and imported using `ZoneProperty:UserViewFactors:bySurfaceName`.
```

Comment:
The internal view factor algorithm is definitely going to have trouble with the geometries that will result here. View3D should not, but it would make sense to add a script or two to simplify that calculation.